From cd545856aa6aab1cb6a5c3c9b71fd06d81ee472a Mon Sep 17 00:00:00 2001
From: Justin Schneck <jschneck@mac.com>
Date: Wed, 4 Aug 2021 17:12:38 -0400
Subject: [PATCH] Symlink the binaries directory

---
 .gitignore | 1 -
 binaries   | 1 +
 2 files changed, 1 insertion(+), 1 deletion(-)
 create mode 120000 binaries

diff --git a/.gitignore b/.gitignore
index e688963..a6c7a8d 100644
--- a/.gitignore
+++ b/.gitignore
@@ -9,7 +9,6 @@ config/at91bootstrap-config/
 build
 debug
 result
-binaries
 *~
 *.orig
 *.o
diff --git a/binaries b/binaries
new file mode 120000
index 0000000..204c670
--- /dev/null
+++ b/binaries
@@ -0,0 +1 @@
+build/binaries
\ No newline at end of file
-- 
2.25.1

