From ed9aea6f4e8b819f893485e2c09cea71514250f7 Mon Sep 17 00:00:00 2001
From: Alex McLain <alex@alexmclain.com>
Date: Sun, 13 Nov 2022 20:09:00 -0800
Subject: [PATCH] Update gpio names

---
 drivers/pinctrl/pinctrl-at91-pio4.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/pinctrl/pinctrl-at91-pio4.c b/drivers/pinctrl/pinctrl-at91-pio4.c
index 706cb8e3e..cfeb69a03 100644
--- a/drivers/pinctrl/pinctrl-at91-pio4.c
+++ b/drivers/pinctrl/pinctrl-at91-pio4.c
@@ -1171,9 +1171,9 @@ static int atmel_pinctrl_probe(struct platform_device *pdev)
 		atmel_pioctrl->pins[i]->line = line;
 
 		pin_desc[i].number = i;
-		/* Pin naming convention: P(bank_name)(bank_pin_number). */
-		pin_desc[i].name = devm_kasprintf(&pdev->dev, GFP_KERNEL, "P%c%u",
-						  bank + 'A', line);
+		/* Pin naming convention: gpioX, where X is the number of the line plus
+			 an offset for the bank that line is on (bank A = 0). */
+		pin_desc[i].name = kasprintf(GFP_KERNEL, "gpio%d", (bank * 32) + line);
 
 		group->name = group_names[i] = pin_desc[i].name;
 		group->pin = pin_desc[i].number;
-- 
2.25.1

